extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -286,6 +290,12 @@ export class AmpStoryConsent extends AMP.BaseElement {
    * @private
    */
   assertAndParseConfig_() {
+    // Validation of the amp-consent config is handled by the amp-consent
+    // javascript.
+    const parentEl = dev().assertElement(this.element.parentElement);
+    const consentScript = childElementByTag(parentEl, 'script');
+    this.consentConfig_ = parseJson(consentScript.textContent);
```

Comment:
Is it possible for this to still be null?

Reply:
Yes but the `amp-consent` would trigger a meaningful error https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/amp-consent.js#L548

Reply:
Is it going to cause unexpected behaviors below where we access keys of `this.consentConfig_`?

Reply:
The first error displayed was ``amp-consent should have (only) one <script> child``, but I still added a test to step out of the method, so we don't pollute the console errors.

PTAL